src/atomate2/abinit/utils/settings.py

```diff
@@ -0,0 +1,45 @@
+"""Settings for abinit flows in atomate2."""
+
+from abipy.flowtk.tasks import TaskManager
+from pydantic import BaseSettings, Field
+
+
+class AbinitAtomateSettings(BaseSettings):
```

Comment:
Can you move these to the normal Atomate2Settings but add an `ABINIT_` prefix to each one? That way all settings can be specified in a single file.

Reply:
done